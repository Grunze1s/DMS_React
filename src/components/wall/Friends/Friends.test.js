import React from 'react';
import Friends from './index';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const setup = (props = {}, state = null) => {
    return (shallow( < Friends {...props}  />)
    )
}

const dummy_data={
    friendList:[{image:'image1.png', name:'name1', status:'online'},{image:'image1.png', name:'name1', status:'away'}]
}
describe('<Friends />', () => {
  

it("Friends render", () => { 
        const wrapper= setup({...dummy_data});
        expect(wrapper).toMatchSnapshot();
  
})
})

