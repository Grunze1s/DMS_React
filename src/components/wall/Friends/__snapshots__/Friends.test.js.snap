// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Friends /> Friends render 1`] = `ShallowWrapper {}`;
